src/components/atoms/button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Button from './button';
+
+describe('Button component', () => {
+	it('it should render snapshot button component', () => {
+		const { asFragment } = render(<Button />);
+		expect(asFragment(<Button />)).toMatchSnapshot();

```

Comment:
@Yasaman-k you could test props too. And check if the props are working as expected. For example, you can test the `className` prop and check if the button has the class given by props. Also for other props too.